### PR TITLE
Show placeholder in Editor when playlist is empty

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Show placeholder in Editor when playlist is empty
Update icon and label 
Bump version

I used a [computed binding](https://polymer-library.polymer-project.org/3.0/docs/devguide/data-binding#annotated-computed) `_shouldNotRender(items)`, so that the placeholder gets recalculated when items change. 

Also revisited the logic for `_itemsChanged()` and added a call to `this._startSchedule()` if playlist is playing. Without this, a playlist that was showing a placeholder would not start playing once items were added. 

## Motivation and Context
Playlist UI epic

## How Has This Been Tested?
Locally and on stage:
https://widgets.risevision.com/staging/components/rise-playlist/editor-placeholder/rise-playlist-bundle.min.js 

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/rise-playlist.js [hunk 1]
```diff
@@ -281,6 +291,10 @@ export default class RisePlaylist extends RiseElement {
     }).forEach(element => {
       this.appendChild(element);
     });
+
+    if (this._isPlaying) {
+      this._startSchedule();
+    }
```
Comment (alex-deaconu) [inline src/rise-playlist.js:297]: @ezequielc not sure I follow why this was needed. We already have this in the `FlattenedNodesObserver` callback. Does that not work correctly? Thanks.

Comment (ezequielc) [reply src/rise-playlist.js]: @alex-deaconu Looks like you are right. For some reason (cache?) my playlist would be blank (i.e. no play event sent to items) without that code. But I double checked now and it works without it. Thanks for bringing this point.

Comment (alex-deaconu) [reply src/rise-playlist.js]: @ezequielc sounds good, I think the `startSchedule() { this._isPlaying = true; ...` fix is what makes it start after item updates. Thanks.

Comment (ezequielc) [reply src/rise-playlist.js]: @alex-deaconu Yeah, I probably changed both at the same time and got tricked by it. Thanks!

**Review by alex-deaconu — APPROVED** [review]
@ezequielc thanks for the update. Tested, looks good 👍

**Pull request merged**